### PR TITLE
Update discriminitor loss function text

### PR DESCRIPTION
**Pull request by anant15**
`log(D(x)) + log(D(G(z)))` --> `log(D(x)) + log(1-D(G(z)))`

### CODE REVIEW

**Comment by holly1238** [comment]
Closing this PR. This issue was fixed and merged in https://github.com/pytorch/tutorials/pull/1051.
